revocation/crl/store.go

```diff
@@ -0,0 +1,200 @@
+package crl
+
+import (
+	"archive/tar"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/notaryproject/notation-core-go/revocation/crl/cache"
+)
+
+const (
+	// BaseCRL is the file name of the base CRL
+	BaseCRL = "base.crl"
+
+	// Metadata is the file name of the metadata
+	Metadata = "metadata.json"
+)
+
+// Store is an interface to store CRL
+type Store interface {
```

Comment:
Just for curious,  since we already have `TrustStore` and `CredentialStore`, shall we rename it to `CRLStore`